c/driver_manager/adbc_driver_manager_test.cc

```diff
@@ -335,4 +338,31 @@ TEST(AdbcDriverManagerInternal, AdbcDriverManagerDefaultEntrypoint) {
     EXPECT_EQ("AdbcProprietaryEngineInit", ::AdbcDriverManagerDefaultEntrypoint(driver));
   }
 }
+
+TEST(AdbcDriverManagerInternal, AdbcParsePath) {
+  // Test parsing a path of directories
+#ifdef __WIN32
+  static const char* const delimiter = ";";
+#else
+  static const char* const delimiter = ":";
+#endif
+
+  std::vector<std::string> paths = {
+      "/usr/lib/adbc/drivers", "/usr/local/lib/adbc/drivers",
+      "/opt/adbc/drivers",     "/home/user/.config/adbc/drivers",
+      "/home/\":foo:\"/bar",
+  };
+
+  std::ostringstream joined;
+  std::copy(paths.begin(), paths.end(),
+            std::ostream_iterator<std::string>(joined, delimiter));
```

Comment:
Apparently, `ostream_iterator` will also append a delimiter at the end (see example):
https://en.cppreference.com/w/cpp/iterator/ostream_iterator.html

Reply:
yea, which is fine for the test because it ensures we also remove empty paths from the returned list